app/src/main/AndroidManifest.xml

```diff
@@ -14,9 +14,9 @@
         android:allowBackup="true"
         android:dataExtractionRules="@xml/data_extraction_rules"
         android:fullBackupContent="@xml/backup_rules"
-        android:icon="@mipmap/ic_launcher"
+        android:icon="@mipmap/ic_launcher_icon"
```

Comment:
@zandre-eng I think we should maintain the previous name, `ic` stands for `icon`

Reply:
It seems there is a conflict with the CommCare Support Library when changing `ic_launcher_icon_foreground.xml` to `ic_launcher_foreground.xml`. I have gone ahead and changed these to `ic_launcher_biometric` ([7b0f999](https://github.com/dimagi/biometric-android/pull/9/commits/7b0f999b826a84f680e40f2d0eee016d25225d22)).